src/constraint/TSR.cpp

```diff
@@ -79,25 +79,28 @@ class TSRSampleGenerator : public SampleGenerator
 
 //=============================================================================
 TSR::TSR(std::unique_ptr<util::RNG> _rng, const Eigen::Isometry3d& _T0_w,
-         const Eigen::Matrix<double, 6, 2>& _Bw, const Eigen::Isometry3d& _Tw_e)
+         const Eigen::Matrix<double, 6, 2>& _Bw, const Eigen::Isometry3d& _Tw_e,
+         const double& _satisfiableTolerance)
     : mT0_w(_T0_w)
     , mBw(_Bw)
     , mTw_e(_Tw_e)
     , mRng(std::move(_rng))
     , mStateSpace(std::make_shared<SE3>())
+    , mSatisfiableTolerance(_satisfiableTolerance)
```

Comment:
Initialize `mSatisfiableTolerance` before `mRng` and `mStateSpace` to resolve the [warning](https://travis-ci.org/personalrobotics/aikido/builds/223337806#L2840).